### DISCUSSION TITLE
radio-live-mg.rtr-vesti.ru

### DISCUSSION THREAD

**Discussion by Dum4G** [iptv-org/community#505] [Show and tell]
rtr and vesti are kind of brands of the past, nowadays Smotri platform is being promoted. However, the source is still working and being actively used although it's now sending smotri certificate and fails to properly set https connection. Do you think it's possible to track proper domain for the source? @BellezaEmporium 

https://radio-live-mg.rtr-vesti.ru/hls/moscow_24/playlist.m3u8

**Comment by Dum4G** [comment]
Fun fact: it should be https://vgtrksmotrim.cdnvideo.ru/hls/moscow_24/playlist.m3u8 from https://smotrim.ru/live/1661 but then you have to have a token or get lost

**Comment by BellezaEmporium** [comment]
https://dvr-player.smotrim.ru/hls/moscow_24/playlist.m3u8

Correct link could be https://dvr-player.smotrim.ru

**Reply by Dum4G** [reply]
You absolutely rock! How do you do this magic?

**Reply by BellezaEmporium** [reply]
I used SecurityTrails in order to check the subdomains.
